addon/styles/addon.scss

```diff
@@ -33,9 +33,11 @@
     flex-direction: row;
     flex: 1;
 
-    > .content {
+    > .main-container {
       flex: 1;
-      margin: 20px 0 0 20px;
+      .frost-object-details__content {
```

Comment:
I didn't really want to name the class this, but since the liquid-outlet adds a few layers between the main-container class and whatever the sub-route's template will be, I didn't see a better option.